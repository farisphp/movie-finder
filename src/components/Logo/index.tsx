function Logo() {
  return (
    <h1 className="text-center text-4xl font-bold text-blue-950">
      Movie Finder
    </h1>
  )
}

export default Logo
