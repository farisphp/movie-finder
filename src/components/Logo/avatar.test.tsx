import { render, screen } from '@testing-library/react'

import Logo from '.'

describe('<Logo />', () => {
  it('should render the logo', () => {
    const { container } = render(<Logo />)

    expect(
      screen.getByRole('heading', { name: /Movie Finder/i, level: 1 })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
