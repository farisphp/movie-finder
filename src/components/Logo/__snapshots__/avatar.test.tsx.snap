// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Logo /> > should render the logo 1`] = `
<h1
  class="text-center text-4xl font-bold text-blue-950"
>
  Movie Finder
</h1>
`;
